user-stories-guidelines.md

```diff
@@ -0,0 +1,87 @@
+
+
+# **CDF User Stories Guidelines**
+
+## Overview
+
+The CDF User Stories Program features real-world use cases from companies embracing the change brought by Continuous Delivery. The goal is to show the impact of adopting Continuous Delivery as a software development practice along with enabling CI/CD technologies, such as the CDF projects.
+
+The stories build narratives around include, but are not limited to:
```

Comment:
```suggestion
The stories we want to build narratives around include, but are not limited to:
```

Reply:
+10  The stories might reflect the role of communities in on-boarding & establishing continuous change to build a stronger connection to the CDF initiatives and importance of building a community 

 The submission form for case study can be updated with the following - 
- Next Steps towards continuous delivery ( to be added by the submitter ) , if that are looking for support for next steps then they can express the interest for support.
- Did they use any of the best practices from CDF